### DISCUSSION TITLE
Is there a repetition here? Is there another use?

### DISCUSSION THREAD

**Discussion by Auase** [musescore/MuseScore#19427] [General]
![leland](https://github.com/musescore/MuseScore/assets/56365735/9426504a-856e-408c-bf1c-8c16362003be)
https://github.com/musescore/MuseScore/blob/0891a471fedb000b6ca642519f64a5b03b71e430/src/framework/fonts/fontsmodule.cpp#L37

**Comment by cbjeukendrup** [comment]
That looks like an undesired repetition indeed. Good catch! (Do you want to remove it, or shall we do it?)

**Reply by Auase** [reply]
You should handle it, after all, there will be fewer audit steps

**Reply by cbjeukendrup** [reply]
Done in #23159
